### PR TITLE
@acjay: Added Artsy setup instructions.

### PR DESCRIPTION
**Pull request by ashfurrow**
Let me know if I can clarify anything.

### CODE REVIEW

**Comment by orta** [comment]
Ace

File: docs/artsy_dev.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+bundle install
+bundle exec pod install
+```
+
+When you run `pod install`, you'll be prompted for API keys. These are stored in the Engineering 1Password vault.
+
+Finally, you can run the tests to verify everything is set up right:
+
+```sh
+fastlane test
```
Comment (acjay) [inline docs/artsy_dev.md]: May want to note that this should be `bundle exec fastlane test` in practice

Comment (ashfurrow) [inline docs/artsy_dev.md]: Ha, easy to forget everyone doesn't use zsh with the `bundler` plugin :P

**Comment by ashfurrow** [comment]
Update'd.

**Comment by acjay** [comment]
:+1:

**Pull request merged**